DOCUMENTATION.md

```diff
@@ -377,25 +377,52 @@ Hull.ready(function(){
 ### `Hull.trackForm(forms, eventName, data)`
 Listen to form submissions and  performs a tracking call
 
-- `forms`: An HTML DOM element or array of such. jQuery supported
```

Comment:
is the jQuery supported a product feature change?

Reply:
Yea. - I don't think we want this anymore